nucypher/characters/lawful.py

```diff
@@ -1388,3 +1345,118 @@ def encrypt_message():
             return Response(json.dumps(response_data), status=200)
 
         return controller
+
+
+class StakeHolder(Staker):
+
+    banner = STAKEHOLDER_BANNER
+
+    class StakingWallet:
```

Comment:
This is also in `characters`? This feels an awful like polluting the namespace here.